Task 1/Array/question.md

```diff
@@ -0,0 +1,24 @@
+Calculate the product of all the elements in an array.
+
+Input:
```

Comment:
Please use markdown heading format for headings.

Something like this

```md
#### Input
```
Renders as 

#### Input

in markdown.